CHANGELOG.md

```diff
@@ -12,15 +12,19 @@ and this project adheres to [Python Versioning](http://legacy.python.org/dev/pep
 * #### NI-DMM
   * #### Added
   * #### Changed
+    * Added default values for some methods.
```

Comment:
Added default values to some parameters.

Reply:
Fixed